console/web3ext/web3ext.go

```diff
@@ -318,6 +318,33 @@ web3._extend({
 			call: 'admin_setMaxSubscriptionPerWSConn',
 			params: 1
 		}),
+		new web3._extend.Method({
+			name: 'startSpamThrottler',
+			call: 'admin_startSpamThrottler',
+			params: 1,
+			inputFormatter: [null]
```

Comment:
Can we call like this without any parameter? How is `config *blockchain.ThrottlerConfig` included?
``` admin.startSpamThrottler()```

Reply:
With this inputFormatter, we can call in both ways of `admin.startSpamThrottler()` and `admin.startSpamThrottler({"activate_tx_pool_size": 1000, ...})`